<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmTX</class>
 <widget class="QMainWindow" name="frmTX">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>510</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>TX</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnApply">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>470</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>应用</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOK">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>381</width>
      <height>306</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tabAxis">
     <attribute name="title">
      <string>局部坐标系</string>
     </attribute>
     <widget class="QLabel" name="lblAngel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>126</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>夹角-XY/YZ/XZ(deg)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtAngel">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>100</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>126</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>绕Z轴旋转(deg)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtRotatebyZ">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>130</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>126</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>坐标轴长度(m)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtAxisLength">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkShowAxis">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>190</y>
        <width>111</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>显示坐标轴</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>126</width>
        <height>25</height>
       </rect>
      </property>
      <property name="locale">
       <locale language="Chinese" country="China"/>
      </property>
      <property name="text">
       <string>原点坐标(m)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtDirection">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtOrigin">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_48">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>126</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>法线方向</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkReverseN">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>190</y>
        <width>81</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>反转N轴</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnStartPick">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>点选</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbxFaces">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>126</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>安装面</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabArrange">
     <attribute name="title">
      <string>阵列天线</string>
     </attribute>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>126</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>V轴平移(m)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>126</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>阵元缩放</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtDistance_Z">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>130</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtDisplayPixel">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkShowArray">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>160</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>显示</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>126</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>导入阵列</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtAntennaFile">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtElementNum">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBrowse">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtDistance_X">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtDistance_Y">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>100</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>126</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>N轴平移(m)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>126</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>U轴平移(m)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>126</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>阵元数</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>离散阵元点</string>
     </attribute>
     <widget class="QTableWidget" name="tbPoints">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>35</y>
        <width>351</width>
        <height>210</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddPoint">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>250</y>
        <width>81</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>添加阵元</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRemovePoint">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>250</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>移除选中行</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>126</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>阵元缩放</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtDisplayPixel_discrete">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>5</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkShowArray_discrete">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>5</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>显示</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabRadio">
     <attribute name="title">
      <string>方向图</string>
     </attribute>
     <widget class="QLabel" name="lblThe_start_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>126</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>原点坐标(m）</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtOrigin_radio">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtRadioFile">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBrowse_radio">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblThe_start_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>126</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>导入方向图</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_43">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>126</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>绕Z轴旋转(deg)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_44">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>126</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>绕Y轴旋转(deg)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtRotate_y">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>130</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtAxisLength_radio">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>220</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_45">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>126</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>缩放比例</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_46">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>126</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>坐标轴宽度(pixel)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtRotate_x">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>100</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblThe_start">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>126</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>绕X轴旋转(deg)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtThickness">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>250</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtRotate_z">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_47">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>126</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>参考坐标轴(m)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtScale">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>190</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkShowAxis_radio">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>220</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>显示</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblAngel_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>126</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>夹角-XY/YZ/XZ(deg)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtAngel_radio">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkShowAntenna_radio">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>190</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>显示</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupPower">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>379</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="lblPower">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>5</y>
       <width>126</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>功率(W)</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblWaveform">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>35</y>
       <width>126</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>信号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPower">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>5</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxWaveform">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>35</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Sinosoid</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Blackman</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Chirp</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gaussian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gaussian dirivative</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hamming</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hanning</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Raised cosine</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="txtName">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>65</y>
       <width>191</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>TX-1</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblTitle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>65</y>
       <width>126</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>名称</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>470</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxAntennaType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>380</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>阵列天线</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>离散阵元</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
